### PR TITLE
EI 751 tech debt remove command controller

### PR DESCRIPTION
**Pull request by Stephek**

### CODE REVIEW

**Review by Pauljgraham — APPROVED** [review]

**Review by David-B-Read — APPROVED** [review]

File: src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -84,5 +58,50 @@ public static IServiceCollection AddCommandService(this IServiceCollection servi
...
         }
+
+        public static IServiceCollection AddNServiceBus(
+            this IServiceCollection serviceCollection,
+            IConfiguration configuration)
+        {
+            var webBuilder = serviceCollection.AddWebJobs(x => { });
+            webBuilder.AddExecutionContextBinding();
+
+            var endpointConfiguration = new EndpointConfiguration("SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers")
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs:69]: hardcoded assembly name

File: src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -84,5 +58,50 @@ public static IServiceCollection AddCommandService(this IServiceCollection servi
...
+
+            if (configuration["ApplicationSettings:NServiceBusConnectionString"].Equals("UseLearningEndpoint=true", StringComparison.CurrentCultureIgnoreCase))
+            {
+                endpointConfiguration
+                    .UseTransport<LearningTransport>()
+                    .StorageDirectory(configuration.GetValue("ApplicationSettings:UseLearningEndpointStorageDirectory",
+                        Path.Combine(
+                            Directory.GetCurrentDirectory().Substring(0, Directory.GetCurrentDirectory().IndexOf("src")),
+                            @"src\SFA.DAS.EmployerIncentives.Functions.TestConsole\.learningtransport")));
+                endpointConfiguration.UseLearningTransport(s => s.AddRouting());
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs:86]: TestConsole ?

**Review by ruslanss — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs (hunk 2 above)
Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs:86]: Yes if "UseLearningEndpoint=true" i.e. a test app is calling.

File: src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs:69]: Nope just an endpoint name.  Could be anything.

**Pull request closed without merging**